ext/greenwich/composer.compile.json

```diff
@@ -5,7 +5,7 @@
       "php-method": "\\Civi\\Compile\\Scss::build",
       "watch-files": ["scss"],
       "scss-files": {"scss/main.scss": "dist/bootstrap3.css"},
-      "scss-includes": ["scss", "extern/bootstrap3/assets/stylesheets"]
+      "scss-includes": ["scss", "extern/bootstrap3/assets/stylesheets", "../../bower_components/select2"]
```

Comment:
@colemanw why is this needed is there an issue with the import if its not there?